src/components/RecipientAddressRow.tsx

```diff
@@ -0,0 +1,34 @@
+import { ExternalLinkIcon } from '@chakra-ui/icons'
+import { HStack, Icon, Link } from '@chakra-ui/react'
+import type { Asset } from '@shapeshiftoss/types'
+
+import { Row } from '@/components/Row/Row'
+import { RawText, Text } from '@/components/Text'
+import { middleEllipsis } from '@/lib/utils'
+
+type RecipientAddressRowProps = {
+  sellAsset: Asset
+  recipientAddress: string | undefined
+}
+
+export const RecipientAddressRow = ({ sellAsset, recipientAddress }: RecipientAddressRowProps) => {
```

Comment:
That is my fault but, `sellAsset` is not required, we can use `explorerLink` as a prop without passing the whole asset so it's even more agnostic!

And we should be good!